labs/303-jaeger-opentracing-traces/README.md

```diff
@@ -0,0 +1,71 @@
+## Importing and Analysing Jaeger Traces in Trace Compass
+
+In this lab, you will learn to use Trace Compass to upload and analyse Jaeger traces, obtained from distributed applications instrumented with OpenTracing API. Jaeger already provides a very good visualization of those traces, but with TraceCompass, those traces can be analyzed along with other system and application traces.
+
+*Pre-requisites*: Have Trace Compass installed and opened. You can follow the [Installing TraceCompass](00-installing-tracecompass.md) lab or read the [TraceCompass web site](https://tracecompass.org) for more information. This lab is not a tutorial about OpenTracing or Jaeger. It supposes that you know about these, but it provides all the necessary material to use Trace Compass with example traces, without having access to a Jaeger infrastructure.
+
+- - -
+
+### Task 1: Tracing with Jaeger
+
+All you need to know to start tracing can be found on Jaeger Tracing website.
+
+[Getting started with Jaeger](https://www.jaegertracing.io/docs/getting-started/)
+
+- - -
+
+### Task 2: Installing Open Tracing Plug-in
+
+You can install the *Open Tracing* trace type in *Tools* -> *Adds-on...*. Check the `Trace Compass opentracing (incubation)` feature and click *Finish*. Follow the instructions on screen.
+
+![AddsOn](screenshots/installPlugIn.png "Trace Compass Install Plug In")
+
+- - -
+
+### Task 3: Fetching Jaeger Traces
+
+Once you have Jaeger running and traced some traces, you can fetch those traces directly into Trace Compass. All you need to do is right click on the Traces folder in the Project Explorer.
+
+![RightClickMenu](screenshots/rightClickMenu.png "Trace Compass Traces Menu")
+
+You will be able to set all the filters you want to apply to your traces request. Once you click the fetch button, you will see a list of traces which you can select the ones you want to import from.
+
+![FetchWindow](screenshots/fetchWindow.png "Trace Compass Fetch Jaeger Window")
+
+Once you click finish, your traces will be imported in your workspace.
+
+![FetchedTraces](screenshots/fetchedTraces.png "Trace Compass Fetched Traces")
+
+You can also open one of the traces in the `traces` directory of this lab.
+
+- - -
+
+### Task 4: Exploring the perspective components
+
+When you open an Open Tracing trace (double click on any imported trace), you should obtain a view that looks like this.
+
+![Perspective](screenshots/perspective.png "Trace Compass Open Tracing Perspective")
+
+Open Tracing perspective contains:
+- Project Explorer: List of your experiments as well as your traces
+- Spans Life View: Time graph representation of the span's relationships
+- Spans List: Information on every span in the trace
```

Comment:
Events Table: Information of every every event as well as every span in the trace. Equivalent of the Jaeger "Spans List".